bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookieWatcherImpl.java

```diff
@@ -62,7 +62,9 @@
             log.error("Interrupted reading bookie list : ", cause);
             return new BKInterruptedException();
         } else {
-            return new MetaStoreException();
+            MetaStoreException mse = new MetaStoreException();
```

Comment:
We should add a constructor which takes directly the cause

Reply:
@eolivelli it seems that I have to change a lot for that. I am new to the project not sure if I should do that. what is the concern regarding using initCause?

Reply:
MetaStoreException already has a constructor which takes the 'cause'.

It is cleaner to use that one and not call initCause

Reply:
@eolivelli I don't think so. MetaStoreException is BKException, BKException doesn't have a constructor with Throwable. but I will try to change it as what you suggested.

Reply:
Oh, don't worry.
I don't have my laptop no
Yesterday I tried to look for the constructor using github web ui and maybe I was wrong and I looked to the wrong class.